### DISCUSSION TITLE
Process to add a new builtin filetype

### DISCUSSION THREAD

**Discussion by chipsenkbeil** [3rd/image.nvim#176] [Q&A] [answered]
Is there documentation that describes what you require and want to see in a pull request to add a new filetype? I want to add in orgmode from https://github.com/nvim-orgmode/orgmode. As the author of https://github.com/chipsenkbeil/org-roam.nvim, I'm fairly well versed in how to use the orgmode plugin's API to find links and determine if they represent images.

My plan was to follow the neorg builtin integration, adjusting to support orgmode instead. Anything I should consider or make note of? Any concerns with me adding this as a contribution?

**Comment by 3rd** [comment, marked as answer]
Hi! All org family members are very welcome <3
It should be straightforward to add support for it; you can copy the neorg integration file, change the filetype and the way it extracts images, and update the integration options (& readme): https://github.com/3rd/image.nvim/blob/master/lua/image/init.lua#L4
I think it could be enabled by default.
Thank you!
